test/autoyast_converter_test.rb

```diff
@@ -93,7 +93,7 @@
     end
   end
 
-  describe ".export" do
+  xdescribe ".export" do
```

Comment:
It is actually my plan to at least create trivial one which allows me to have trivial case where is one parrition with "/" on it.